prediction_market_agent/agents/microchain_agent/nft_treasury_game/scripts/reset_balance_anvil.py

```diff
@@ -40,10 +43,11 @@ def get_token_owner(token_id: int, web3: Web3) -> ChecksumAddress:
     return nft_contract.owner_of(token_id=token_id, web3=web3)
 
 
-def is_treasury_empty(rpc_url: str, balance_min_threshold: xDai = xDai(0.1)) -> bool:
-    w3 = Web3(Web3.HTTPProvider(rpc_url))
-    balance = get_balances(SimpleTreasuryContract().address, web3=w3)
-    return balance.xdai < balance_min_threshold
+def is_treasury_empty(rpc_url: str) -> bool:
+    return (
+        get_nft_game_status(web3=Web3(Web3.HTTPProvider(rpc_url)))
+        == NFTGameStatus.finished
+    )
```

Comment:
I would refactor this to `is_game_finished` or similar, because that's what we are checking for.